src/components/Player/Pages/Counts/Counts.jsx

```diff
@@ -13,14 +14,18 @@ import styles from './Counts.css';
 const Counts = ({ playerId, counts, getPlayerCounts }) => (
   <div>
     <TableFilterForm submitAction={getPlayerCounts} id={playerId} page="counts" />
-    <div className={styles.countsContainer}>
-      {Object.keys(counts).map((key, index) => (
-        <div key={index} className={styles.countTable}>
-          <TableContainer title={deSnake(key)} />
-          <Table paginated columns={playerCountsColumns} data={counts[key].list} />
-        </div>
-      ))}
-    </div>
+    <Row>
+      <div className={styles.countsContainer}>
+        {Object.keys(counts).map((key, index) => (
+          <Col xs={12} sm={12} md={6} lg={4} key={index}>
+            <div className={styles.countTable}>
+              <TableContainer title={deSnake(key)} />
+              <Table paginated columns={playerCountsColumns} data={counts[key].list} />
```

Comment:
I just tried this method on the matches table and it won't work though. The table breaks at around 780px width. We still need to allow scrolling on tables if they get broken or have some other solution for tables with more than 3 small columns.
![](http://i.imgur.com/mEpY2R1.png)